wormhole/file_transport.go

```diff
@@ -263,19 +294,14 @@ func (t *fileTransport) connectDirect(otherTransit *transitMsg) (net.Conn, error
 		}
 	}
 
-	return conn, nil
+	return conn.successChan, nil
```

Comment:
same comment as above. The return type is `net.Conn`, but to someone who reads the code and not look up the return type, it looks as if this returns a `channel` which is not the case.